errors/index.js

```diff
@@ -1,22 +1,20 @@
-
-exports.handleCustomErrors = (err, req,res,next) => {
-    if(err.status && err.msg) {
-        res.status(err.status).send({msg: err.msg}); 
-    } else {
-        next(err);
-    }
-}
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
 
```

Comment:
This is spot on, and allows you to reuse this middleware with different status codes and messages over time, well done.